act-rules-format.bs

```diff
@@ -235,158 +257,198 @@ When all expectations are true for a test target, the test target `passed` the r
 
 Even within a [=composite rule=], some [=atomic rules=] may not always be applicable. This leaves one fewer solution for passing that particular composite rule. To support users of ACT Rules in properly defining the [accessibility support baseline](https://www.w3.org/TR/WCAG-EM/#step1c) in their test scenarios, an ACT Rule SHOULD include a warning if there are significant accessibility support concerns known about a rule.
 
 
-ACT Data Format (Output Data) {#output}
-=======================================
-
-With ACT Rules, it is important that data coming from different sources can be compared. By having a shared vocabulary, accessibility data that is produced by different auditors can be compared and, where necessary, aggregated. Therefore, every ACT Rule MUST express the output in a format that has all of the features described in the ACT Data Format.
-
-Rules are tested in two steps. Firstly, the applicability is used to find a list of [Test Targets](#output-test-target) (elements, tags or other "components") within the web page or other [test subject](#output-test-subject). Then each test target is tested to see if all of the [expectations](#expectations) are true. This will give the outcome for each test target. For contextual information, the output data must also include [test subject](#output-test-subject) and the [rule identifier](#rule-identifier).
-
-This will mean that every time a rule is executed on a page, it will return a set with zero or more results, each of which MUST have at least the following properties:
+Evaluation Instructions {#evaluation-instructions}
+========================
 
-- [Rule Identifier](#rule-identifier) (test)
-- [Test Subject](#output-test-subject) (Web page)
-- [Test Target](#output-test-target) (pointer)
-- [Outcome](#output-outcome) (`Passed`, `Failed`, or `Inapplicable`)
+An ACT Rule MUST include evaluation instructions. Evaluation instructions describes how applicability and expecations can be used to determine (non-)conformance to the [listed accessibility requirements](#accessibility-requirements). Evaluation instructions MUST indicate how `passed`, `failed` and `inapplicable` [=outcomes=] can be determined for each [=test subject=], when evaluating the rule. 
```

Comment:
The first sentence is already covered by section 4 "Rule Structure"; the second sentence is already covered by section 11 "Expectations"; and the third sentence is already covered by the definition of "outcomes". I think we can lose this entire paragraph. It re-defines existing normative requirements and definitions, which is very confusing.